### PR TITLE
aplay: change if statement to else if in xrun function

### PR DESCRIPTION
**Pull request by JSFamily**
Honestly, It does not affect behavior in runtime. But I made this patch to help understanding the source code.

### CODE REVIEW

File: aplay/aplay.c [hunk 1]
```diff
@@ -1613,8 +1613,7 @@ static void xrun(void)
 	if ((res = snd_pcm_status(handle, status))<0) {
 		error(_("status error: %s"), snd_strerror(res));
```
Comment (dbaluta) [inline aplay/aplay.c:1614]: Care to explain in the commit message why this change is needed? What is the problem and why your change fixes it.

Comment (JSFamily) [reply aplay/aplay.c:1614]: Honestly, It does not affect behavior in runtime. But I push this commit to help understanding the source code.

**Comment by perexg** [comment]
Fixed only the second condition (missing newline) in 986a1bd3d2eebd41a2925969826fca870b2cd330 .
